api/gateway/state.go

```diff
@@ -9,17 +9,12 @@ import (
 	"github.com/cosmos/cosmos-sdk/types"
 	"github.com/gorilla/mux"
 
-	"github.com/celestiaorg/celestia-node/blob"
 	"github.com/celestiaorg/celestia-node/state"
 )
 
 const (
-	balanceEndpoint            = "/balance"
-	submitTxEndpoint           = "/submit_tx"
-	submitPFBEndpoint          = "/submit_pfb"
-	queryDelegationEndpoint    = "/query_delegation"
-	queryUnbondingEndpoint     = "/query_unbonding"
-	queryRedelegationsEndpoint = "/query_redelegations"
+	balanceEndpoint  = "/balance"
```

Comment:
I'm a bit concerned with allowing reads for balance without passing an addr (which would be just a `Balance` query and would access the signers account addr and check its balance)

Reply:
This PR handles that correctly - balance w/o an address writes an error